### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by joacomiguens**
## Summary

- Create navbar.

- Update book-list to fit with navbar justification.

### Scrolling Percentage: 0%
image

### Scrolling Percentage: More than 0%

image

## Trello Card

You can find it [here](https://trello.com/c/csrzW1fy/31-navbar)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,89 +8,97 @@
...
   <body>
-    <div class="container">
-      <div class="detail">
-        <div class="detail__cover">
-          <img src="./assets/book-cover.png" alt="cover of the book" class="detail__cover__img" />
-          <img src="./assets/badge.png" alt="badge of the book" class="detail__cover__badge" />
+    <header>
+      <nav class="nav">
+        <div class="nav__content">
+          <img src="assets/LogoWolox.png" class="nav__content__logo" />
```
Comment (ebedoyaalzate) [inline index.html]: Add alt

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (joacomiguens) [reply index.html]: Done 😄

File: index.html [hunk 2]
```diff
@@ -8,89 +8,97 @@
...
           <div class="book__info">
             <h3 class="book__info__title">Título del libro</h3>
             <h4 class="book__info__author">Autor del libro</h4>
           </div>
         </div>
       </div>
+      <div class="detail">
+        <div class="detail__cover">
+          <img src="./assets/book-cover.png" alt="cover of the book" class="detail__cover__img" />
+          <img src="./assets/badge.png" alt="badge of the book" class="detail__cover__badge" />
```
Comment (ebedoyaalzate) [inline index.html:81]: Remember correct the previous PR's errors in the rest of PR's

Comment (ebedoyaalzate) [inline index.html]: Remember correct the previous PR's suggestions in the rest of PR's.

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

**Review by ebedoyaalzate — APPROVED** [review]

File: scss/_book-list.scss [hunk 3]
```diff
@@ -2,17 +2,17 @@
...
   padding: 32px 32px 20px 32px;
 
   &-list {
     display: flex;
     flex-wrap: wrap;
-    justify-content: center;
+    justify-content: space-between;
     margin: 40px 0;
     max-width: $desktop-width;
     & > * {
```
Comment (mcallegari10) [inline scss/_book-list.scss]: Use `nth-child` instead 😄 . We usually don't use the * operator

File: scss/_book-list.scss [hunk 4]
```diff
@@ -2,17 +2,17 @@
   @include card-shadow;
   display: flex;
   flex-direction: column;
-  max-width: 20%;
+  max-width: calc(25% - 40px);
```
Comment (mcallegari10) [inline scss/_book-list.scss]: Why not set a specific size and then `width: 100%`?

File: scss/_styles.scss [hunk 5]
```diff
@@ -2,10 +2,15 @@
   font-family: 'Roboto', sans-serif;
 }
 
+body {
```
Comment (mcallegari10) [inline scss/_styles.scss]: To add a style to the body aside from the equalizer is not really good practice. Try to set it to the container

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: scss/_book-list.scss (hunk 3 above)
Comment (joacomiguens) [reply scss/_book-list.scss]: Done 😄

File: scss/_book-list.scss (hunk 4 above)
Comment (joacomiguens) [reply scss/_book-list.scss]: I change percentages to px values and use `width: 100%`

File: scss/_styles.scss (hunk 5 above)
Comment (joacomiguens) [reply scss/_styles.scss]: Done 😄

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**